### ISSUE TITLE
TODO: Define Transfer Format (delta index?)

### ISSUE THREAD

**Issue by Psifour** [casey/runestone#9]
"Decide whether or not ID should be a delta, which would be more compact, but would require that transfers be encoded in increasing order of asset ID."

**Comment by summraznboi** [comment]
I like it, only because it generalizes using 0 ID to continue reusing the previous rune ID as opposed to making 0 be a special case. (And in practice, people won't really care to handle multiple rune types in a single transaction, so its more about protocol consistency more than anything else).

**Comment by Psifour** [comment]
Definitely dependent on #8. I lean towards the blockheight + txindex option there which would naturally preclude this option, but would also mean that we lose the natural byte-savings that we gain with delta-based transfers (unless we were to keep it in some form that would likely lead to edge-cases).

**Comment by casey** [comment]
Id, amount, and output can all be deltas. Id is always a delta, amount is a delta if the ID is zero, and output is a delta if the amount is zero.

**Comment by casey** [comment]
Closing in favor of #60.
